### DISCUSSION TITLE
Missing Polygon Representation in Martin with Maplibre

### DISCUSSION THREAD

**Discussion by orttak** [maplibre/martin#657] [Q&A]
I noticed that when I zoom in to the same spatial area, the number of polygons generated varies between zoom levels. For example, at zoom level 13, I get 3 polygons, but when I zoom in to level 14, I only see 2 polygons, and the previous 3 polygons are no longer visible.

As a newbie in Martin, I'm unsure if this behavior is expected or if there might be some configuration or setting that I need to adjust. I would appreciate any guidance or insights regarding this issue.

Furthermore, I would like to know if there is a more detailed version of Martin available that provides additional documentation or explanations about its behavior and usage.

Additionally, I would like to inquire if there is an active community around Martin, such as a Gitter or Discord group, where I can seek support, discuss questions, and interact with other users of the tool.

zoom level = x
![image](https://github.com/maplibre/martin/assets/28723856/f492db73-2877-495f-8cd8-90dab0bf61f1)

zoom level = x+1
![image](https://github.com/maplibre/martin/assets/28723856/cd235dd1-2934-4fd5-9a9e-66c74126c584)

**Comment by orttak** [comment]
I should ask this in discussion, Sorry, for that. I'd be happy if you delete this

**Reply by nyurik** [reply]
I moved it to discussions

**Comment by nyurik** [comment]
@orttak you may want to try [debugging steps](https://github.com/maplibre/martin#debugging) to see the actual query being ran. To make logging a bit more "sane", first figure out which tile is causing problems, and try to get just that one tile at different zoom levels.  This way you will see what PostgreSQL query is being made. Martin has almost no logic when making these queries - it simply asks PG for all geometries in a given area (computed with [ST_TileEnvelope](https://postgis.net/docs/ST_TileEnvelope.html)), and lets PG encode those geometries as tiles.

**Reply by orttak** [reply]
I solved it with max_feature_count for reading from database.

When I try to read mbtiles which I create from qgis, I got below log and mbtiles show like below image.
Is there any spesific rule for producing mbtiles for martin or setting for martin's config file?


```
[2023-05-10T11:45:33Z INFO  actix_web::middleware::logger] 192.168.0.1 "GET /parcelmbtiles/11/1079/662 HTTP/1.1" 200 1521628 "http://127.0.0.1:5500/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0" 0.052381
[2023-05-10T11:45:34Z INFO  sqlx::query] SELECT tile_data from tiles …; rows affected: 0, rows returned: 1, elapsed: 671.384ms
    
    SELECT
      tile_data
    from
      tiles
    where
      zoom_level = ?
      AND tile_column = ?
      AND tile_row = ?
    
[2023-05-10T11:45:34Z INFO  actix_web::middleware::logger] 192.168.0.1 "GET /parcelmbtiles/11/1080/661 HTTP/1.1" 200 16166566 "http://127.0.0.1:5500/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0" 0.807047
```

![image](https://github.com/maplibre/martin/assets/28723856/98409bd1-3075-4c9f-808e-adb11183c1f3)
